### PR TITLE
refactor: replace moment with dayjs

### PR DESCRIPTION
**Pull request by faustbrian**
Replaces all occurrences of `moment` with `dayjs`.

### CODE REVIEW

File: src/renderer/mixins/formatter.js [hunk 1]
```diff
@@ -34,7 +34,20 @@ export default {
...
+
+      if (userLanguage === 'en-us') {
+        userLanguage = 'en'
+      }
+
+      try {
+        require(`dayjs/locale/${userLanguage}`)
+      } catch {
+        userLanguage = 'en'
+      }
```
Comment (dated) [inline src/renderer/mixins/formatter.js]: More generic:

```suggestion
      try {
        require(`dayjs/locale/${userLanguage}`)
      } catch {
        try {
          userLanguage = userLanguage.split('-')[0]
          require(`dayjs/locale/${userLanguage}`)
        } catch {
          userLanguage = 'en'
        }
      }
```

Comment (dated) [reply src/renderer/mixins/formatter.js]: This will also handle locales like `de-DE`, `it-IT` ...

Comment (faustbrian) [reply src/renderer/mixins/formatter.js]: Also more expensive due to how require works inside of electron.

Comment (dated) [reply src/renderer/mixins/formatter.js]: Gimme a minute, I'll boil it down to one `require`

Comment (dated) [reply src/renderer/mixins/formatter.js]: Update above

**Comment by alexbarnsley** [comment]
`moment` is used in `src/renderer/services/crypto/transaction-signer.js` since a different PR was merged

**Pull request merged**